bindata/v3.11.0/kube-apiserver/recovery-config.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: kubecontrolplane.config.openshift.io/v1
+kind: KubeAPIServerConfig
+apiServerArguments:
+  storage-backend:
+  - etcd3
+  storage-media-type:
+  - application/vnd.kubernetes.protobuf
+servingInfo:
+  bindAddress: 0.0.0.0:7443
+  bindNetwork: tcp4
+  certFile: /etc/kubernetes/static-pod-resources/serving-ca.crt
+  keyFile: /etc/kubernetes/static-pod-resources/serving-ca.key
+  clientCA: /etc/kubernetes/static-pod-resources/serving-ca.crt
+storageConfig:
+  keyFile: /etc/kubernetes/static-pod-resources/etcd-client.key
+  certFile: /etc/kubernetes/static-pod-resources/etcd-client.crt
+  ca: /etc/kubernetes/static-pod-resources/etcd-serving-ca-bundle.crt
+  storagePrefix: openshift.io
+  urls:
+  - "https://localhost:2379"
+
+# Make hypershift happy
+authConfig:
+  oauthMetadataFile: ""
+  requestHeader:
+    clientCA: /etc/kubernetes/static-pod-resources/serving-ca.crt
+serviceAccountPublicKeyFiles:
+- /etc/kubernetes/static-pod-resources/serving-ca.crt
+kubeletClientInfo:
+  ca: /etc/kubernetes/static-pod-resources/serving-ca.crt
+  certFile: /etc/kubernetes/static-pod-resources/serving-ca.crt
+  keyFile: /etc/kubernetes/static-pod-resources/serving-ca.key
+aggregatorConfig:
+  proxyClientInfo:
+    certFile: /etc/kubernetes/static-pod-resources/serving-ca.crt
```

Comment:
this should be loadable

Reply:
I though we don't want to set up aggregation for recovery apiserver